bode/bode/models/task_relation.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+from enum import Enum
+
+from sqlalchemy.dialects.postgresql import ENUM, UUID
+
+from bode.app import db
+
+
+class RelationType(Enum):
+    Dependent = "DEPENDENT"
+    Interchangable = "INTERCHANGABLE"
+    Subtask = "SUBTASK"
+
+    @classmethod
+    def list(cls):
+        return list(map(lambda c: c.value, cls))
```

Comment:
I think that a list comprehension will be faster here:
```suggestion
        return [c.value for c in cls]
```